reference/data/models/pumpsettings/pumpsettings.v1.yaml

```diff
@@ -150,6 +150,44 @@ allOf:
         type: string
         minLength: 1
         maxLength: 100
+      sleepSchedules:
+        type: array
+        minItems: 0
+        maxItems: 10
+        items:
+          type: object 
+          properties:
+            enabled: 
+              type: boolean
+            days:
+              type: array
+              uniqueItems: true
+              minItems: 1
+              maxItems: 7
+              items:
+                type: string
+                enum:
+                - Sunday
+                - Monday
+                - Tuesday
+                - Wednesday
+                - Thursday
+                - Friday
+                - Saturday
+            start:
+              type: number
+              format: integer
+              minimum: 0
+              maximum: 86400
+              description: Start time in seconds, offset from midnight (e.g. 11pm is 82800)
+            end:
+              type: number
+              format: integer
+              minimum: 0
+              maximum: 86400
+              description: end time in seconds, offset from midnight (e.g. 7am is 25200)
```

Comment:
Is it possible for `end` <= `start`? AFAIK there's no cross-field validation in OAS3 so it's mainly idle curiosity.

Or perhaps more importantly, how does one express something like "start at 11pm on Tuesday, end at 7am Wednesday"? Something like `{ enabled: true, days: [ "Tuesday", "Wednesday" ], start: 8200, end: 25200 }`? But that might be ambiguous?

Reply:
Yes, `end` can be smaller than `start`. In fact, that is the most likely case. To express your example, you would just do:

`{ enabled: true, days: [ "Tuesday" ], start: 82800, end: 25200 }`